### ISSUE TITLE
[tool_crash] ProcessException: Access denied.  Command: C:\..\AppData\Local\Temp\flutter_gradle_wrapper.599db19d-889e-11ea-9d18-9cb6d0039cef\gradlew.bat -b

### ISSUE THREAD

**Issue by angelbuzon** [flutter/flutter#55742]
## Command
```
flutter --no-color --no-version-check precache
```

## Steps to Reproduce
1. ...
2. ...
3. ...

## Logs
ProcessException: ProcessException: Acceso denegado.

  Command: C:\Users\angel\AppData\Local\Temp\flutter_gradle_wrapper.599db19d-889e-11ea-9d18-9cb6d0039cef\gradlew.bat -b D:\\flutter\packages\flutter_tools\gradle\resolve_dependencies.gradle --project-cache-dir C:\Users\angel\AppData\Local\Temp\flutter_gradle_wrapper.599db19d-889e-11ea-9d18-9cb6d0039cef resolveDependencies
```

```
```
[✓] Flutter (Channel beta, v1.17.0-3.2.pre, on Microsoft Windows [VersiÃ³n 10.0.18362.778], locale es-ES)
    • Flutter version 1.17.0-3.2.pre at D:\flutter
    • Framework revision 2a7bc389f2 (6 days ago), 2020-04-21 20:34:20 -0700
    • Engine revision 4c8c31f591
    • Dart version 2.8.0 (build 2.8.0-dev.20.10)

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
    • Android SDK at D:\Android\android-sdk
    • Platform android-29, build-tools 29.0.0
    • ANDROID_HOME = D:\Android\android-sdk
    • Java binary at: D:\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b03)
    • All Android licenses accepted.

[✓] Android Studio (version 3.5)
    • Android Studio at D:\Android\Android Studio
    • Flutter plugin version 41.1.2
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b03)

[!] Connected device
    ! No devices available

! Doctor found issues in 1 category.

```

## Flutter Application Metadata
No pubspec in working directory.

**Comment by jmagman** [comment]
@zanderso I think you mentioned this was a duplicate of another issue when Flutter is installed on a shared drive in Windows?

**Comment by zanderso** [comment]
Hmm, no. I think this might be similar issue to https://github.com/flutter/flutter/issues/56070. That is, we might not be able to reliably execute scripts out of the system temporary directory.

**Comment by zanderso** [comment]
@angelbuzon I've landed #56706 on master, and if you're able to give it a shot that would be very helpful. Thanks for filing the issue!

**Comment by zanderso** [comment]
Presumed fixed by https://github.com/flutter/flutter/pull/56706
